src/components/Grid/Grid.tsx

```diff
@@ -0,0 +1,93 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import React from 'react';
+import { DimensionUnity, SpacingValue } from '@utilities/dimensions';
+import { ContainerHTMLElement } from 'src/types/elements';
+import { merge } from '@utilities/merge';
+import { Box } from '../Box/Box';
+
+export type GridProps = {
+    column?: number;
+    spacingX?: SpacingValue;
+    spacingY?: SpacingValue;
+    children?: React.ReactNode;
+    width?: `${number}${DimensionUnity}`;
+    rowHeight?: `${number}${DimensionUnity}`;
+    padding?: SpacingValue;
+    margin?: SpacingValue;
+    bg?: string;
+    color?: string;
+    'data-test-id'?: string;
+    as?: ContainerHTMLElement;
+};
+
+export const CONTAINER_TEST_ID = 'fondue-grid';
+
+const spacingMap = {
```

Comment:
You can create a map constant for gap, gap-x and gap-y constants in the `@utilities/dimentions` module, like the ones created for padding and margin.